src/zmalloc.c

```diff
@@ -164,7 +164,7 @@ void zfree_no_tcache(void *ptr) {
 
 /* Try allocating memory and zero it, and return NULL if failed.
  * '*usable' is set to the usable size if non NULL. */
-void *ztrycalloc_usable(size_t size, size_t *usable) {
+__attribute__((flatten)) void *ztrycalloc_usable(size_t size, size_t *usable) {
     /* Possible overflow, return NULL, so that the caller can panic or handle a failed allocation. */
     if (size >= SIZE_MAX/2) return NULL;
     void *ptr = calloc(1, MALLOC_MIN_SIZE(size)+PREFIX_SIZE);
```

Comment:
that can't cause `calloc` to be inlined (being a library import), right? (which is desired).
i.e. just wanna make sure we only inline the small functions of zmalloc.c with one another, and not something big from outside.